### DISCUSSION TITLE
Individual match count for all terms using multiword search

### DISCUSSION THREAD

**Discussion by rohan-gosavi** [stephanrauh/ngx-extended-pdf-viewer#2583] [Q&A]
Is there a way to get the match count for each of the terms using multiple word search?
for example: "blood,bmi,diabetes" then can we get match count for each as [blood = 10, bmi = 4, diabetes = 50]?

**Comment by stephanrauh** [comment]
Yes, this makes sense. I don't know why this didn't cross my mind before. Thanks for pointing this out!

I've opened a ticket dedicated to the issue.

**Reply by rohan-gosavi** [reply]
Thanks for addressing @stephanrauh

**Reply by victorDECLAU** [reply]
I adressed this issue in versions < 17.4.10 using `matchesColors` from the result of the findMultiple() method. 
With `matchesColors` it is possible to deduce those counters :)

**Reply by stephanrauh** [reply]
It seems the cross-reference is missing. Here is the ticket: #2586

**Reply by stephanrauh** [reply]
Let me elaborate Victor's solution: Until version 18, the PDF viewer had an extension to the find bar. It allowed you do find multiple search terms simultaneously, marking the search result in different colors. The programmatic API returned an array called `matchesColors`. This array was meant to be used internally, but as a side effect, it allowed you to count the frequency of every search term.

Unfortunately, I had to abandon this solution because I ended up in merge hell. After one or two years, I decided to ditch my improvements in favor of the improvements of the base library.
